### DISCUSSION TITLE
Data Science

### DISCUSSION THREAD

**Discussion by Jyotika32** [microsoft/Data-Science-For-Beginners#147] [Ideas]
How to start learning Data-Science  for first time ?

**Comment by jlooper** [comment]
We recommend working through this curriculum to get started

**Comment by Holy452** [comment]
Is there any free course on Data Science online?

**Reply by jlooper** [reply]
You’re viewing one now. This repo contains 20 free lessons.

**Reply by kraliq0** [reply]
Molecular Soul

**Reply by kraliq0** [reply]
> You’re viewing one now. This repo contains 20 free lessons.

Molecular Soul physics

**Reply by yeganeh74** [reply]
"Yes! You can use this repository. Also, Kaggle offers very useful courses."
